pages/bakeries/[id].js

```diff
@@ -86,7 +93,7 @@ export default function BakeryDetailPage({ bakeryData }) {
             cost={item.cost}
             productId={item.productId}
             key={item.productId}
-            onAddToCart={addToCart}
+            onAddToCart={() => onAddToCart(item, setCart, cart)}
```

Comment:
In your component `BakeryMenu` you work with the prop `onAddToCart` and can just passed in down here
```suggestion
            onAddToCart={onAddToCart}
```